### ISSUE TITLE
Option to pass in query shape without boolean

### ISSUE THREAD

**Issue by bret-hubbard** [dupski/json-to-graphql-query#14]
TL;DR I would like the option to be able to pass in an object for a query and have all properties on the query whether the value of a property is truthy or not.

I am using yup on a project where I am already defining my schema and I am using the default of the schema for the form state in react and then ultimately for the mutation query.

I don't like having to define the schema in so many different places and in this case the query is the same shape as the mutation. It would be nice to be able to use the default object from yup like I am to define the default form state in react to define the query.

I am willing to do the work and submit a PR for this if it something you can see value in.

Thanks!

**Comment by dupski** [comment]
Hey @bret-hubbard 

Yep happy to accept a PR for this. Maybe we could add an `ignoreBooleans` option? which stops the library from excluding falsy keys.

Bare in mind that obviously if your data has object keys / values, they would end up as nodes in the resulting query.

Please add tests and run `npm test` and `npm run build` before submitting a PR. Cheers! :)
